Cargo.toml

```diff
@@ -57,8 +57,9 @@ num_cpus = "1"
 protobuf = "2.20.0"
 rand = "0.8"
 regex = "1.4"
-sentry = { version="0.23.0", features = ["curl"]}
-sentry-backtrace = "0.23.0"
+# pin to 0.19: https://github.com/getsentry/sentry-rust/issues/277
```

Comment:
We might want to add a note here saying something like "Need to pin this to 0.19 until on-premises sentry server is updated", just to prevent future issues.